torch/_appdirs.py

```diff
@@ -515,7 +515,7 @@ def _get_win_folder_from_registry(csidl_name):
         _winreg.HKEY_CURRENT_USER,
         r"Software\Microsoft\Windows\CurrentVersion\Explorer\Shell Folders",
     )
-    dir, type = _winreg.QueryValueEx(key, shell_folder_name)
+    dir, _ = _winreg.QueryValueEx(key, shell_folder_name)
```

Comment:
If we don't want to lose info, replaceing it `_type` etc is also valid and fine.

Reply:
Dang, that's an interesting idea I didn't know, and I think I'm going to go back and do that for at least some of these variables.

Thanks for cluing me in, @Skylion007.